src/asynqp/channel.py

```diff
@@ -37,7 +37,7 @@ def __init__(self, id, synchroniser, sender, basic_return_consumer, queue_factor
         self.reader = reader
 
     @asyncio.coroutine
-    def declare_exchange(self, name, type, *, durable=True, auto_delete=False, internal=False):
+    def declare_exchange(self, name, type, *, durable=True, auto_delete=False, internal=False, arguments={}):
```

Comment:
`arguments={}` sets off my ["mutable default argument alarm bell"](http://stackoverflow.com/questions/1132941/least-astonishment-in-python-the-mutable-default-argument). I also think that perhaps a `**kwargs`-based API might be more pleasant to use as a client (it would also take care of the mutable default argument issue).